articles/i2p-docker-services/BasicStaticeepSite.md

```diff
@@ -125,9 +125,9 @@ ARG WEBSITE=website
 ENV WEBSITE=$WEBSITE
 RUN apk update
 RUN apk add darkhttpd
-COPY $WEBSITE /car/www/localhost/htdocs/$WEBSITE
+COPY $WEBSITE /var/www/localhost/htdocs/$WEBSITE
```

Comment:
Haha, didn't even notice that typo, good pick :+1:

Reply:
Yeah because COPY makes sure the directory exists, and because I copied the typo to the following line, it didn't even break the tutorial. I only noticed it because I re-used part of it.